### PR TITLE
Issue 33: adding pull request and issue templates

### PR DESCRIPTION
**Pull request by rsk2**
For issue #33

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### 👨‍💻 What's being changed:
+
+<!-- List all the proposed changes in your PR. -->
+
+### Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+<!-- Follow the below conventions to check the box -->
```
Comment (unnikrishnan2002) [inline .github/pull_request_template.md:16]: In the check boxes add one more box to ensure that user has got the issue assigned before creating the PR

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### ✔️ Check List (Check all the applicable boxes) 
+
+<!-- Mark all the applicable boxes -->
+
+- [ ] I claimed any associated issue(s) and they are not someone else's
+- [ ] I have looked at documentation to ensure I made any revisions correctly
+- [ ] I tested my changes locally to ensure they work
+- [ ] The title of my pull request is a short description of the requested changes.
+
```
Comment (unnikrishnan2002) [inline .github/pull_request_template.md:36]: Add a section for screenshots after the new changes

**Review by unnikrishnan2002 — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (rsk2) [reply .github/pull_request_template.md:16]: Line 32 is similar: 

`I claimed any associated issue(s) and they are not someone else's`

Do you want me to change the wording of this?

Comment (unnikrishnan2002) [reply .github/pull_request_template.md:16]: Nvm it is okay

**Review by unnikrishnan2002 — APPROVED** [review]

**Pull request merged**